src/controllers/venue.controller.ts

```diff
@@ -96,6 +96,73 @@ export const getVenue = async (req: Request, res: Response) => {
   }
 };
 
+export const getVenues = async (req: Request, res: Response) => {
+  let targetVenues=[];
+
+  const userId = req.query?.userId;
+  console.log('here')
+  targetVenues = await Venue.aggregate([
+    {
+      $addFields: {
+        reactions: {
+          $arrayToObject: {
+            $map: {
+              input: REACTION_EMOJIS,
+              as: 'emoji',
+              in: {
+                k: '$$emoji',
+                v: {
+                  count: {
+                    $size: {
+                      $filter: {
+                        input: '$reactions',
+                        cond: {
+                          $eq: ['$$this.emoji', '$$emoji'],
+                        },
+                      },
+                    },
+                  },
+                  didReact: {
+                    $anyElementTrue: {
+                      $map: {
+                        input: {
+                          $filter: {
+                            input: '$reactions',
+                            cond: {
```

Comment:
Can you change the endpoint to only select with a query param of count? 
‘venues/?count=10’ would return 10 of the venues etc.

We'll have to change it like this later so we might as well do it now